server/jetstream_cluster.go

```diff
@@ -4352,28 +4352,18 @@ func (js *jetStream) processClusterCreateConsumer(ca *consumerAssignment, state
 	// Check if we already have this consumer running.
 	o := mset.lookupConsumer(consumer)
 
-	if !alreadyRunning {
```

Comment:
Is it safe to remove this logic here?

Reply:
Yes, `createRaftGroup` knows if the Raft node already exists and only checks peer differences.